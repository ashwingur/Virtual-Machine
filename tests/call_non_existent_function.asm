FUNC LABEL 0
    PRINT VAL 2
    CAL VAL 3
    CAL VAL 6
    CAL VAL 7
    RET
FUNC LABEL 3
    PRINT VAL 3
    RET
FUNC LABEL 6
    PRINT VAL 6
    RET