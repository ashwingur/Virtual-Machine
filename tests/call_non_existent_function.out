2
3
6
Function does not exist.
